rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -530,15 +539,15 @@ public LRAStatus getStatus(URI uri) throws WebApplicationException {
 
         try {
             client = getClient();
-            response = client.target(base)
+            response = client.target(coordinatorUrl)
                 .path(String.format(STATUS_PATH, LRAConstants.getLRAUid(uri)))
                 .request()
                 .get();
 
             if (response.getStatus() == NOT_FOUND.getStatusCode()) {
-                String responseContent = response.readEntity(String.class);
-                String errorMsg = "The requested LRA it '" + lraId + "' was not found and the status can't be obtained"
-                    + (responseContent != null ? ": " + responseContent : "");
+                String responseEntity = response.hasEntity() ? response.readEntity(String.class) : "";
+                String errorMsg = "The requested LRA it '" + lraId + "' was not found and the status can't be obtained, "
+                        + ", response content: " + responseEntity;
```

Comment:
this will be `...obtained, , response content:...`

Reply:
+1